.github/workflows/crowdin-download.curriculum.yml

```diff
@@ -248,9 +252,39 @@ jobs:
           GH_BRANCH: 'i18n-sync-curriculum'
           GH_MESSAGE: 'chore(i18n,learn): processed translations'
 
-      # Generate PR #
+      # Validate the Download #
```

Comment:
We'll want to run these before the commit, since `npm run format` will write to the files.

Reply:
🤦🏽‍♂️ - You are right.